Backend/app/rider-platform/rider-app/Main/spec/API/MultiModal.yaml

```
@@ -138,4 +141,14 @@ apis:
\ No newline at end of file
+        type: JourneyStatus
+
+  - POST:
+      endpoint: multimodal/{journeyId}/order/{legOrder}/switchTaxi
+      params:
+        journeyId: Id Journey
+        legOrder: Int
+      request:
+        type: SwitchTaxiReq
+      response:
+        type: JourneyInfoResp
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)